docs/recipes/plugin/lighthouse-plugin-preload-as/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "lighthouse-plugin-preload-as",
+  "private": true,
+  "main": "./plugin-config.js"
```

Comment:
recipe-custom-config.js

Reply:
> recipe-custom-config.js

oh, this one is correct. The custom config is what you'd use with `--config-path` to run lighthouse with the plugin. The plugin's `main` is the actual plugin config file.

Reply:
> Hmmm.. weird to have two things that work together both called a config. about how `index.js` or `plugin-index.js`? `plugin-dfn.js`...

I'd actually rather have it stay as `plugin-config.js` and (eventually) get rid of the other one in this recipe.

Most of the time you'll either be writing a plugin or using a plugin in your config, not writing both at the same time, so it shouldn't cause confusion, and it makes sense to keep calling it a "plugin config" because it's pretty much identical to the main config except more locked down.

We need to add a `--plugin` cli flag, but at that point you won't need the `recipe-custom-config.js` file, which should clear up the confusion here.

Reply:
> a "plugin config" because it's pretty much identical to the main config except more locked down.

ehhh. IMO the Lighthouse concept of 'config' is very defined. And I think it'll be more confusing to use the term for two things that are similar but slightly different. It's not like our docs for plugins should say "go look at Lighthouse config, but only these parts."

Looking over at [eslint plugins](https://eslint.org/docs/developer-guide/working-with-plugins), they basically just call this a 'plugin'. No 'definition' or 'config' term. That sgtm. I still favor `index.js` but `plugin.js` seems ok.

Reply:
> but `plugin.js` seems ok.

sure